pymc3/tests/test_distributions_random.py

```diff
@@ -1737,21 +1737,18 @@ def test_density_dist_with_random_sampleable_handcrafted_success_fast(self):
                 random=rvs,
                 wrap_random_with_dist_shape=False,
             )
-            trace = pm.sample(100, cores=1)
-
-        samples = 500
-        size = 100
```

Comment:
This test looks suspiciously like some asserts were accidentally deleted at some point in the past.
This is why I prefer to put a `pass` at the end of tests!